### PR TITLE
[WIP] Add ability to extract gnomad AFs for PRS.

### PR DESCRIPTION
**Pull request by cristinaetrv**
Add extraction of gnomad AFs for the 5 superpopulations using query of target dataset annotation in order to enable weighing PRS by gnomad AFs.

### CODE REVIEW

File: python/python/bystro/prs/preprocess_for_prs.py [hunk 1]
```diff
@@ -61,6 +62,47 @@ def _load_genetic_maps_from_feather(map_path: str) -> dict[str, pd.DataFrame]:
...
+        "(gnomad.genomes.AF_amr:>0)",
+        "(gnomad.genomes.AF_eas:>0)",
+        "(gnomad.genomes.AF_nfe:>0)",
+        "(gnomad.genomes.AF_sas:>0)"
+    ]
+    full_query = allele_frequency_filters + finalized_loci
+    return f'"{" || ".join(full_query)}"'
+
+    
+def _extract_af_and_loci_overlap(score_loci: set, target_dataset_index: str, user: str) -> pd.DataFrame:
```
Comment (akotlar) [inline python/python/bystro/prs/preprocess_for_prs.py:86]: user is CachedAuth

Comment (akotlar) [reply python/python/bystro/prs/preprocess_for_prs.py:86]: and then we want to take an optional opensearch_cluster_config, which is a dict

Comment (akotlar) [reply python/python/bystro/prs/preprocess_for_prs.py:86]: and both user and opensearch_cluster_config should be optional; we just need to check that one is not None (and has values)

**Review by akotlar — CHANGES_REQUESTED** [review]
A few small changes then LGTM

**Comment by cristinaetrv** [comment]
Included in #529

**Pull request closed without merging**